### ISSUE TITLE
Build ARIA labels component

### ISSUE THREAD

**Issue by cehfisher** [cehfisher/a11y-style-guide#71]
Creating a ticket to track progress on ARIA labels component.

**Comment by scottaohara** [comment]
what does this refer to specifically?

like the various ways one could implement the ```aria-label```, or ```aria-labelledby``` attributes for form controls, links, buttons, landmarks, etc?

**Comment by cehfisher** [comment]
hmm why the thumbs down @rbenson?

@scottaohara I want to include some "best practices" for ARIA. i truly believe that `no ARIA is better than bad ARIA` but people do use it. I want to add some ways to use it right

**Comment by scottaohara** [comment]
so then it sounds like the definition of 'component' was at play here.

understood and agree.  good area to explore

**Comment by rbenson** [comment]
@cehfisher  Because this wouldn't follow the [rules of ARIA](https://www.w3.org/TR/aria-in-html/#rule1)

>If you can use a native HTML element [HTML51] or attribute with the semantics and behaviour you require already built in, instead of re-purposing an element and adding an ARIA role, state or property to make it accessible, then do so.

Given the guide is missing various a11y basics, such as #63, focusing on ARIA, or an ARIA-first approach, goes against best practices and the ARIA spec itself.

**Comment by scottaohara** [comment]
so then you're suggesting that it not even be addressed rather than provide examples of when it would be appropriate to use, @rbenson?

Also, as you pointed out there is still quite a bit to do here, so feel free to help out!  @cehfisher has been great to work with on PRs

**Comment by cehfisher** [comment]
yep, sounds like we all are on the same page! it might be best to remove them as actual "components" though as that is addressed within each specific example. i'm going to close this ticket and the other ARIA one, and move those into the "resource" section.
